django/pyproject.toml

```diff
@@ -53,6 +56,7 @@ flake8-pie = "^0.6.1"
 flake8-printf-formatting = "^1.1.0"
 flake8-pytest-style = "^1.3.0"
 pep8-naming = "^0.11.1"
+watchdog = {extras = ["watchmedo"], version = "^1.0.2"}
```

Comment:
Inconsistent formatting
```suggestion
watchdog = { version = "^1.0.2", extras = ["watchmedo"] }
```

Reply:
I'd leave it as is or add an autoformatter, since this is how poetry added it automatically and doing formatting by hand seems a bit excessive. Know of any autoformatters for toml?

Reply:
Possibly [Prettier Toml Plugin](https://www.npmjs.com/package/prettier-plugin-toml)

It's annoying because poetry generates TOML that doesn't follow the formatting standards shown in every example and I think it even generates inconsistent TOML